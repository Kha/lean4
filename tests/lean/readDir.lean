#eval repr <$> IO.readDir "Reformat"
