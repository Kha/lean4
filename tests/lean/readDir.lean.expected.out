#[{ root := "Reformat", fileName := "Input.lean" }]
